egs/bentham/v1/path.sh

```diff
@@ -0,0 +1,9 @@
+export KALDI_ROOT=`pwd`/../../..
+[ -f $KALDI_ROOT/tools/env.sh ] && . $KALDI_ROOT/tools/env.sh
+export PATH=$PWD/utils/:$KALDI_ROOT/tools/openfst/bin:$PWD:$PATH
+[ ! -f $KALDI_ROOT/tools/config/common_path.sh ] && echo >&2 "The standard file $KALDI_ROOT/tools/config/common_path.sh is not present -> Exit!" && exit 1
+. $KALDI_ROOT/tools/config/common_path.sh
+
+export LD_LIBRARY_PATH=$KALDI_ROOT/tools/openfst/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/home/dpovey/libs:$LD_LIBRARY_PATH
```

Comment:
we can remove local changes from the pull request. we can make it similar to kaldi/egs/swbd or kaldi/egs/wsj. can you please make a similar change in kaldi/egs/iam/path.sh